WalletWasabi/WabiSabi/Client/AliceClient.cs

```diff
@@ -0,0 +1,101 @@
+using Microsoft.Extensions.Hosting;
+using NBitcoin;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using WalletWasabi.Bases;
+using WalletWasabi.Helpers;
+using WalletWasabi.Logging;
+using WalletWasabi.WabiSabi.Backend.PostRequests;
+using WalletWasabi.WabiSabi.Models;
+
+namespace WalletWasabi.WabiSabi.Client
+{
+	public class AliceClient : BackgroundService, IAsyncDisposable
+	{
+		private Guid AliceId { get; }
+		private Guid RoundId { get; }
```

Comment:
This properties could be `public` too because they are immutable and exposing them cannot have side effect.

Reply:
ACK